src/Orleans.Clustering.Kubernetes/Orleans.Clustering.Kubernetes.csproj

```diff
@@ -27,7 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="KubernetesClient" Version="10.0.31" PrivateAssets="All" />
+    <PackageReference Include="KubernetesClient" Version="10.0.31" />

```

Comment:
<img width="701" alt="image" src="https://github.com/OrleansContrib/Orleans.Clustering.Kubernetes/assets/4714040/ebd67d7c-fa6d-44e4-a1cf-55eb35781083">

I think we should upgrade this as well. Not sure if there are too many breaking changes...

Reply:
There was this PR, that's why I didn't upgrade:
https://github.com/OrleansContrib/Orleans.Clustering.Kubernetes/pull/69

But makes sense to me. Will include that upgrade here.